As nix chroot environment does not have the 'root' as owner, we have to disable
the "owner=root" tar parameters when doing these tar files.
They are built at openoffice build time.
diff --git a/sysui/desktop/slackware/makefile.mk b/sysui/desktop/slackware/makefile.mk
index 3342aca..49679b1 100644
--- a/sysui/desktop/slackware/makefile.mk
+++ b/sysui/desktop/slackware/makefile.mk
@@ -100,7 +100,7 @@ $(MISC)/$(TARGET)/usr/share/applications/ :
 
 $(MISC)/$(TARGET)/empty.tar :
 	@$(MKDIRHIER) $(@:d)/empty
-	@tar -C $(MISC)/$(TARGET)/empty --owner=root --group=root --same-owner -cf $@ .
+	@tar -C $(MISC)/$(TARGET)/empty  -cf $@ .
 
 
 # --- packaging ---------------------------------------------------
@@ -112,7 +112,7 @@ $(MENUFILES) : $(MISC)/$(TARGET)/empty.tar
 	-$(RM) -r $(MISC)$/$(@:b)
 	dmake $(MISC)$/$(@:b)$/usr/share/applications $(MISC)$/$(@:b)$/install$/slack-desc $(MISC)$/$(@:b)$/install$/doinst.sh
 	@$(COPY) $(MISC)/$(TARGET)$/empty.tar $@.tmp
-	@tar -C $(MISC)/$(@:b) --owner=root --group=root --same-owner --exclude application.flag -rf $@.tmp install usr opt
+	@tar -C $(MISC)/$(@:b) --exclude application.flag -rf $@.tmp install usr opt
 	@gzip < $@.tmp > $@
 	@$(RM) $@.tmp
 	$(RM) -r $(MISC)$/$(@:b)
